cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinOrganization.java

```diff
@@ -67,17 +67,23 @@ public boolean equals(Object o) {
         if (this == o) {
             return true;
         }
-        if (!(o instanceof CristinOrganization)) {
+        if (o == null || getClass() != o.getClass()) {
             return false;
         }
+
         CristinOrganization that = (CristinOrganization) o;
-        return Objects.equals(getInstitution(), that.getInstitution()) && Objects.equals(
-            getInstitutionUnit(), that.getInstitutionUnit());
+
+        if (!Objects.equals(institution, that.institution)) {
+            return false;
+        }
+        return Objects.equals(institutionUnit, that.institutionUnit);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(getInstitution(), getInstitutionUnit());
+        int result = institution != null ? institution.hashCode() : 0;
```

Comment:
why are you doing this? is it not enough with Objects.hash?

Reply:
This is me using wrong intellij template